### DISCUSSION TITLE
Configure renovate to ignore dependencies versions from specific Maven repo

### DISCUSSION THREAD

**Discussion by Abdallah-Abdelazim** [renovatebot/renovate#28130] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

We have an Android app. Renovate is opening PRs to update the `org.hamcrest:hamcrest-integration` to version `1.4-atlassian-1`. This version comes from Atlassian's public Maven repo (`https://packages.atlassian.com/maven/repository/public`) and not from the official MavenCentral.
We need to configure renovate to stop such PRs.

Reading [the renovate Configuration Options docs](https://docs.renovatebot.com/configuration-options), I've come up with this package rule but I don't know if it's correct or not:\
```
"packageRules": [
    {
      "matchPackageNames": ["org.hamcrest:hamcrest-integration"],
      "excludeRepositories": ["https://packages.atlassian.com/maven/repository/public"],
      "enabled": false
    }
  ]
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by secustor** [comment, marked as answer]
`excludeRepositories` does not work that way, it is used if you want to NOT set a setting for a specific repository. 

What you probably want is a hostRule with `enabled: false`
https://docs.renovatebot.com/configuration-options/#hostrules

**Reply by Abdallah-Abdelazim** [reply]
Thanks for your swift answer.
So I should add this to my `renovate.json` config file, right? Will it work in disabling the Atlassian repo at this specified URL?
```
"hostRules": [
    {
      "matchHost": "https://packages.atlassian.com/maven/repository/public",
      "enabled": false
    }
  ]
```

**Reply by secustor** [reply]
Yes and yes.
